### ISSUE TITLE
Synchronizing directories => 1 hour time difference (summer/winter time)

### ISSUE THREAD

**Issue by michaelhutter** [doublecmd/doublecmd#470]
When I synchronize two directories (e. g. from my local hard disc (ext4) and a external hard disc (NTFS)) then sometimes there is a difference of 1 hour between the files which I copied some months before.
So I see hundreds of files which DoubleCommander thinks are different on both sides, but actually aren't :-(
May be this is because of the time change (summer/winter time) combined with the usage of two different file systems?
In FreeCommander (which only runs in Windows) I have two options in the sync dialog which are missing in DoubleCommander:
"**ignore 1 hour time delay**" and "up to 2 Sec. ignore"
These options/checkboxes would be very helpful if they would be integrated.

**Comment by Skif-off** [comment]
[NtfsHourTimeDelay](http://doublecmd.github.io/doc/en/configxml.html)?

**Comment by alexx2000** [comment]
This option is Windows only. Because I don't know how to determine ntfs file system when it mounted via ntfs-3g.

**Comment by michaelhutter** [comment]
@Skif-off The documentation of the hidden option NtfsHourTimeDelay shows me that it is exactly what I looked for.
I changed the option to True while DoubleCommander was not running. After restarting DC the problem is still there. The synchronising dialog (Alt+s) shows many files as different which only have 1 hour difference (I don't see them when I activate the checkbox "ignore date").
**It seems to me that the option [NtfsHourTimeDelay](http://doublecmd.github.io/doc/en/configxml.html) does not work / has a bug.**
On the left side I have ext4, on the right side I have NTFS file system.

@alexx2000  I am working on Linux! I knew this problem already from windows (where I have FreeCommander with the option "ignore 1 hour time delay" to solve this issue). But it is the same using Linux.

**Comment by alexx2000** [comment]
Yes, it is not a bug, I don't know how to get this information. So this option does nothing under Linux.

**Comment by michaelhutter** [comment]
Think about a option which ignores exactly 1 hour difference **independent** of any file system. That would help me a lot.
